test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -19,8 +19,9 @@ define([
             expect(cell.renderIcon).not.toBeDefined();
             const cellWidget = BulkImportCell.make({cell, initialize: true});
             expect(cellWidget).toBeDefined();
-            expect(cell.getIcon).toBeDefined();
-            expect(cell.renderIcon).toBeDefined();
+            ['getIcon', 'renderIcon', 'maximize', 'minimize'].forEach( (method) => {
+                expect(cell[method]).toBeDefined();
+            });
```

Comment:
Kind of a crap test, I know! From my investigations, the mock cell doesn't have the appropriate DOM structure to be able to test whether or not classes are added or removed from various DOM nodes.